core/frontend/package.json

```diff
@@ -99,6 +98,7 @@
     "@itwin/object-storage-core": "~1.4.0",
     "@itwin/core-i18n": "workspace:*",
     "@itwin/core-telemetry": "workspace:*",
+    "@itwin/webgl-compatibility": "workspace:*",
```

Comment:
mind removing the direct dep we have on this in display-test-app?

Reply:
Removed.

Reply:
weird, ci is complaining about missing dep on it now, but technically it would be installed anyway as a transitive dep. im unable to repro the build failure locally, but i guess since we do have an import from this pkg we should leave it in. going to drop your last commit